SafeNeighbourhood/Classes/NeighborhoodData.cs

```diff
@@ -0,0 +1,81 @@
+﻿// <auto-generated />
+//
+// To parse this JSON data, add NuGet 'Newtonsoft.Json' then do:
+//
+//    using QuicktypeNeighborhood;
+//
+//    var neighborhoodData = NeighborhoodData.FromJson(jsonString);
+
+namespace QuicktypeNeighborhood
+{
+    using System;
+    using System.Collections.Generic;
+
+    using System.Globalization;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+
+    public partial class NeighborhoodData
+    {
+        [JsonProperty("neighborhood")]
+        public string Neighborhood { get; set; }
+
+        [JsonProperty("wfdb")]
+        [JsonConverter(typeof(ParseStringConverter))]
+        public long Wfdb { get; set; }
```

Comment:
The Wfdb attribute contains the total number of "Cleanliness" complaints filed for each neighborhood